planet/__init__.py

```diff
@@ -29,7 +27,6 @@
     'OrdersClient',
     'order_request',
     'reporting',
-    'Planet',
```

Comment:
What if we kept `Planet` exported at the top-level (e.g. `from planet import Planet` and only move the individual API classes to `planet.sync`? Thinking through this a little bit more since the discussion yesterday, `from planet.sync import Planet` could be harder for users to discover without referencing the docs, and `sync` is probably not as meaningful to end users as it is for us.

`from planet import *`  may be cluttered but I think it's appropriate for Planet/AsyncPlanet (our entrypoints) to get first pick at this namespace.

Otherwise this looks like a good approach to address the docs issue - just thinking out loud about the best way for users to discover how to start using the SDK.

Reply:
> from planet import * may be cluttered but I think it's appropriate for Planet/AsyncPlanet (our entrypoints) to get first pick at this namespace.

Good idea @stephenhillier, I updated this PR to promote the sync Planet class wrapper to the top level namespace.  So it can be imported via `from planet import Planet`, and `from planet import *`.  I still have the sync Planet class wrapper living in `./sync/client.py`.